mix.exs

```diff
@@ -2,7 +2,7 @@ defmodule BuildkiteTestCollector.MixProject do
   use Mix.Project
   @moduledoc false
 
-  @version "0.2.2"
+  @version "0.3.0"
```

Comment:
@gchan I'm not downgrading, I was just looking for a way to do a patch release with `mix git_ops.release` and thought I had to rollback the version in here to do it. But it just does minor releases by default so now we're bumped to `0.3.0`

Reply:
I realised my comment wasn't relevant as I looking at the individual commits where I noticed it went down to 0.2.0. I deleted my comment after realising the subsequent commit bumps it up to 0.3.0!